Tests/verify_end_to_end_purchase.py

```diff
@@ -0,0 +1,175 @@
+import time
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+
+from Pages.home_page import HomePage
```

Comment:
I would put Pages in lower case as it is a package. `pages.home_page`

Reply:
Pages package renamed to pages per naming conventions.